api/envoy/config/cluster/v3/cluster.proto

```diff
@@ -508,6 +508,18 @@ message Cluster {
       google.protobuf.UInt32Value hash_balance_factor = 2 [(validate.rules).uint32 = {gte: 100}];
     }
 
+    enum EndpointWarmingPolicy {
+      NO_WAIT = 0;
+      WAIT_FOR_FIRST_PASSING_HC = 1;
+    }
+
+    // Configuration for slow start mode.
+    // [#next-free-field: 3]
+    message SlowStartConfig {
+      google.protobuf.UInt32Value slow_start_window = 1;
```

Comment:
Please add comment to fields

Reply:
thanks for review @htuch, i will fix api+docs once PR is in more mature state.